test/abstract_syntax_tree.coffee

```diff
@@ -889,80 +1002,211 @@ test "AST as expected for BooleanLiteral node", ->
 #     body:
 #       type: 'Block'
 
-# test "AST as expected for Op node", ->
-#   testExpression '1 <= 2',
-#     type: 'Op'
-#     operator: '<='
-#     originalOperator: '<='
-#     flip: no
-#     first:
-#       value: '1'
-#     second:
-#       value: '2'
-
-#   testExpression '1 is 2',
-#     type: 'Op'
-#     operator: '==='
-#     originalOperator: 'is'
-#     flip: no
-
-#   testExpression '1 // 2',
-#     type: 'Op'
-#     operator: '//'
-#     originalOperator: '//'
-#     flip: no
-
-#   testExpression '1 << 2',
-#     type: 'Op'
-#     operator: '<<'
-#     originalOperator: '<<'
-#     flip: no
-
-#   testExpression 'new Old',   # NOTE: `new` with params is a `Call` node.
-#     type: 'Op'
-#     operator: 'new'
-#     originalOperator: 'new'
-#     flip: no
-#     first:
-#       value: 'Old'
-
-#   testExpression '-> await 2',
-#     type: 'Code'
-#     isAsync: yes
-#     body:
-#       type: 'Op'
-#       operator: 'await'
-#       originalOperator: 'await'
-#       first:
-#         type: 'NumberLiteral'
-#         value: '2'
-
-#   testExpression '-> yield 2',
-#     type: 'Code'
-#     isGenerator: yes
-#     body:
-#       type: 'Op'
-#       operator: 'yield'
-#       originalOperator: 'yield'
-#       first:
-#         type: 'NumberLiteral'
-#         value: '2'
-
-# test "AST as expected for In node", ->
-#   testExpression '1 in 2',
-#     type: 'Op'
-#     operator: 'in'
-#     originalOperator: 'in'
-#     flip: no
-#     first:
-#       value: '1'
-#     second:
-#       value: '2'
-
-#   testExpression 'for x in 2 then',
-#     type: 'For'
-#     range: no
-#     pattern: no
+test "AST as expected for Op node", ->
+  testExpression '1 <= 2',
+    type: 'BinaryExpression'
+    operator: '<='
+    left:
+      type: 'NumericLiteral'
+      value: 1
+    right:
+      type: 'NumericLiteral'
+      value: 2
+
+  testExpression '1 is 2',
+    type: 'BinaryExpression'
+    operator: 'is'
+    left:
+      type: 'NumericLiteral'
+      value: 1
+    right:
+      type: 'NumericLiteral'
+      value: 2
+
+  testExpression '1 // 2',
+    type: 'BinaryExpression'
+    operator: '//'
+    left:
+      type: 'NumericLiteral'
+      value: 1
+    right:
+      type: 'NumericLiteral'
+      value: 2
+
+  testExpression '1 << 2',
+    type: 'BinaryExpression'
+    operator: '<<'
+    left:
+      type: 'NumericLiteral'
+      value: 1
+    right:
+      type: 'NumericLiteral'
+      value: 2
+
+  testExpression 'new Old',   # NOTE: `new` with params is a `Call` node.
```

Comment:
...more evidence? (I didn't write this comment)

Reply:
Not sure what you mean?